.gitignore

```diff
@@ -3,7 +3,7 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
```

Comment:
The `.venv/` directory is correctly ignored, but ensure that this is the intended virtual environment directory for your project. If your virtual environment is located elsewhere, adjust the path accordingly.